libs/payments/cart/src/lib/cart.error.ts

```diff
@@ -82,3 +82,39 @@ export class CartInvalidStateForActionError extends CartError {
     });
   }
 }
+
+export class CartTotalMismatchError extends CartError {
+  constructor(
+    cartId: string,
+    cartAmount: number,
+    invoiceAmount: number,
+    cause?: Error
```

Comment:
nit, doesn't look like `cause` gets used, and it's not required by `CartError`, so this can be removed.

Also applies to CartEligiblityMismatchError